openwisp_notifications/websockets/handlers.py

```diff
@@ -1,8 +1,28 @@
 from asgiref.sync import async_to_sync
 from channels import layers
+from django.core.cache import cache
+from django.utils.timezone import now, timedelta
 
 from openwisp_notifications.api.serializers import NotFound, NotificationListSerializer
 
+from ..swapper import load_model
+
+Notification = load_model('Notification')
+
+
+def user_in_notification_storm(user):
+    in_notification_storm = cache.get(f'ow-noti-storm-{user.pk}', False)
+    if in_notification_storm:
+        return True
+    qs = Notification.objects.filter(recipient=user)
+    if qs.filter(timestamp__gte=now() - timedelta(seconds=10)).count() >= 6:
+        in_notification_storm = True
+    elif qs.filter(timestamp__gte=now() - timedelta(seconds=60)).count() >= 30:
+        in_notification_storm = True
+    if in_notification_storm:
+        cache.set(f'ow-noti-storm-{user.pk}', True, 60)
+    return in_notification_storm
```

Comment:
Now that it's working better, I think we can centralize these numbers in a dictionary and allow this dict to be overridden via settings if needed, what do you think?